application/templates/pages/about/performance.md

```diff
@@ -24,9 +24,9 @@ We check data sources for issues and help data owners improve the quality. This
 
 <!-- ### ?? out of ?? quality score -->
 
-62 data sources on the platform with no issues (47%). 8 fixed last month (+15%).
+47 data sources on the platform with no issues (34%). <!-- 8 fixed last month (+15%). --> 
```

Comment:
Is there a reason you've taken the number fixed out? It could be something I'm not up-to-date on, so useful to know!

Reply:
Only because this number fluctuates so much each week - was 59 last week and could easily go back up again next week. Can leave it in if you think it's more transparent

Reply:
OK, thanks. Ideally it shouldn't fluctuate, it should go down consistently or only fluctuate by marginal amounts. Having said that, I know the reason it fluctuates and that the new performance database will help us count issues day by day. Happy to drop it for now.